src/serac/numerics/functional/tensor.hpp

```diff
@@ -390,6 +390,13 @@ SERAC_HOST_DEVICE zero& get(zero& x)
   return x;
 }
 
+/** @brief let `zero` be accessed like a tuple */
+template <int i>
+SERAC_HOST_DEVICE zero get(zero)
+{
+  return zero{};
```

Comment:
For some reason, this produces obscure `get` resolution errors.

Reply:
interesting, I'd like to hear more info about this at some point